src/test/fuzz/integer.cpp

```diff
@@ -124,4 +127,68 @@ void test_one_input(const std::vector<uint8_t>& buffer)
         (void)GetScriptForDestination(destination);
         (void)IsValidDestination(destination);
     }
+
+    {
+        CDataStream stream(SER_NETWORK, INIT_PROTO_VERSION);
+
+        uint256 deserialized_u256;
+        stream << u256;
+        stream >> deserialized_u256;
+        assert(u256 == deserialized_u256);
+
```

Comment:
Could also assert that the data stream is empty after each operation?

Reply:
Oh, of course! Good idea. Now fixed :)